modules/calc/src/main/java/com/opengamma/strata/calc/ColumnHeader.java

```diff
@@ -0,0 +1,412 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.calc;
+
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.Set;
+
+import org.joda.beans.Bean;
+import org.joda.beans.BeanBuilder;
+import org.joda.beans.BeanDefinition;
+import org.joda.beans.ImmutableBean;
+import org.joda.beans.JodaBeanUtils;
+import org.joda.beans.MetaProperty;
+import org.joda.beans.Property;
+import org.joda.beans.PropertyDefinition;
+import org.joda.beans.impl.direct.DirectFieldsBeanBuilder;
+import org.joda.beans.impl.direct.DirectMetaBean;
+import org.joda.beans.impl.direct.DirectMetaProperty;
+import org.joda.beans.impl.direct.DirectMetaPropertyMap;
+
+import com.opengamma.strata.basics.currency.Currency;
+import com.opengamma.strata.calc.config.Measure;
+import com.opengamma.strata.calc.config.ReportingCurrency;
+
+/**
+ * Provides access to the column name and measure in the grid of results.
+ * <p>
+ * {@link CalculationRunner} provides the ability to calculate a grid of results
+ * for a given set targets and columns. This defines the columns in the results.
+ */
+@BeanDefinition(builderScope = "private")
+public final class ColumnHeader implements ImmutableBean {
+
+  /**
+   * The column name.
+   * <p>
+   * This is the name of the column, and should be unique in a list of columns.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final ColumnName name;
+  /**
+   * The measure that was calculated.
+   * <p>
+   * This defines the calculation that was performed, such as 'PresentValue' or 'ParRate'.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final Measure measure;
+  /**
+   * The currency of the result.
+   * <p>
+   * If the measure is {@linkplain Measure#isCurrencyConvertible() currency-convertible}
```

Comment:
I think "can be automatically converted to a different currency" would be more understandable than "currency-convertible".